src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,169 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DOUBLING_FACTOR = 2;
+    private Node<K,V>[] table;
+    private int threshold;
+    private int size;
+
+    @SuppressWarnings({"unchecked"})
+    public MyHashMap() {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the constructor.